licenses.d2iq.yaml

```diff
@@ -484,10 +484,10 @@ resources:
       - license_path: LICENSE
         ref: v${image_tag%-d2iq.0}
         url: https://github.com/weaviate/weaviate
-  - container_image: ghcr.io/kube-logging/fluentd:v1.16-full-build.122
+  - container_image: ghcr.io/kube-logging/fluentd:v1.16-4.8-filters-build.155
     sources:
       - url: https://github.com/fluent/fluentd
-        ref: ${image_tag%-full-build.122}
+        ref: ${image_tag%-filters-build.155}
```

Comment:
```suggestion
        ref: ${image_tag%-4.8-filters-build.155}
```

Reply:
Hi @mhrabovcin,
I made the necessary changes in the code but still, the CI tests are failing, please suggest on the same.